app/src/main/java/com/github/freeman/bootcamp/games/guessit/GameOptionsActivity.kt

```diff
@@ -332,13 +344,41 @@ fun GameOptionsBackButton() {
     }
 }
 
+@OptIn(ExperimentalMaterial3Api::class)
+@Composable
+private fun PasswordInput(password: MutableState<String>) {
+    Row(
+        verticalAlignment = Alignment.CenterVertically,
+        modifier = Modifier
+            .padding(8.dp)
+            .testTag("passwordInput")
+    ) {
+        TextField(
+            value = password.value,
+            onValueChange = {
+                password.value = it
+            },
+            modifier = Modifier
+                .testTag("passwordInputTextField"),
+            placeholder = { Text(PASSWORD_PLACEHOLDER) },
+            colors = TextFieldDefaults.textFieldColors(
+                focusedIndicatorColor = Color.Transparent,
+                unfocusedIndicatorColor = Color.Transparent
```

Comment:
Since this is not a sensitive task, I suggest that we don't allow users to set complicated passwords. This will make it easier for users to join private lobbies, so that they don't spend too much time entering the password.
This can be simply done by using the keyboardOptions parameter of the TextField and setting the KeyboardType to NumberPassword (as an example).